docbrown/db/src/graphdb.rs

```diff
@@ -700,6 +713,56 @@ mod db_tests {
         assert_eq!(both_expected, both_actual);
     }
 
+    #[quickcheck]
+    fn vertices_window_test()  {
+        // make 4 graphs with different num of partitions
+        let graph_one = GraphDB::new(1);
+        let graph_three = GraphDB::new(3);
```

Comment:
You may probably want to compare results from two different graphs here.

Reply:
ok done